mem-03-D.xml

```diff
@@ -95,7 +95,6 @@
       <column name="definition">you-him/her/it/them</column>
       <column name="definition_de">du-ihn/sie/es/sie (pl.)</column>
       <column name="definition_fa">شما او / او / آن / آنها [AUTOTRANSLATED]</column>
-      <column name="definition_sv">du-honom/henne/det/dem</column>
```

Comment:
This shouldn't have been deleted.

Reply:
I have no idea how that happened. I'll try to be more careful next time.